### PR TITLE
chore: double push to magma artifactory is removed

### LINKED ISSUE #14531: Remove the double push again
When the artifactory migration is finished, we should only push to linuxfoundation.jfrog.io.

### PR DESCRIPTION
**Pull request by nstng**
## Summary

This change removes the double push to the magmacore artifactory, i.e., only the push to the LF artifactory remains.

For build_all some additional refactoring was necessary:
* parsing the response of the agw upload for firebase needed to be adapted for the jfrog cli upload - thanks to @jheidbrink for support in refactoring this

## Test Plan

CI - but only really on master

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: .github/workflows/build_all.yml [hunk 1]
```diff
@@ -176,50 +162,42 @@ jobs:
         env:
           DEBIAN_META_INFO: deb.distribution=focal-ci;deb.component=main;deb.architecture=amd64
         run: |
+          set -euo pipefail
```
Comment (sebathomas) [inline .github/workflows/build_all.yml:169]: Are we sure we didn't somehow rely on not failing in the past?

File: .github/workflows/build_all.yml [hunk 2]
```diff
@@ -176,50 +162,42 @@ jobs:
...
-          if [[ "$PUBLISH_ERROR" != "true" ]]; then
-            ARTIFACTS=$(echo $ARTIFACTS | jq '. += {"valid":true}')
-          fi
+            "(*).deb" magma-packages-test/pool/focal-ci/{1}.deb)
+
+          # Mapping response to output (for firebase)
+          echo "Response:"
+          echo $RESPONSE | jq
+          echo "Output (for firebase):"
+          ARTIFACTS=$(echo $RESPONSE | jq '{"valid": (.status=="success"), "packages": [.files[] | .target]}')
```
Comment (sebathomas) [inline .github/workflows/build_all.yml:204]: :+1:

File: .github/workflows/build_all.yml (hunk 1 above)
Comment (nstng) [reply .github/workflows/build_all.yml:169]: Will summarize my understanding in order to find an answer ... The step does three things
* find version of created magma debian artifact for integration test workflow
* upload created artifacts
* create firebase output from upload response

The step can be seen as failed (imho) if no magma artifact was created or if the upload fails - but the question is what failed mean here: I assume we always want to create the firebase output, i.e., the current restriction might be to hard. Suggestion: lets create the firebase output as failed before the `set -euo pipefail` and override it in the follow up steps.

Comment (nstng) [reply .github/workflows/build_all.yml:169]: changed - small test if the override of the output works as expected: https://github.com/nstng/magma/actions/workflows/aaa_test_output.yml (I did not find this in the documentation)

File: .github/workflows/build_all.yml (hunk 2 above)
Comment (nstng) [reply .github/workflows/build_all.yml:204]: kudos to @jheidbrink - my initial approach to build the output was more procedural-ly

**Review by sebathomas — APPROVED** [review]

**Review by tmdzk — APPROVED** [review]

**Review by jheidbrink — APPROVED** [review]

**Review by LKreutzer — APPROVED** [review]

**Pull request merged**